pep-0704.rst

```diff
@@ -0,0 +1,170 @@
+
+Such a documentation page should be made available in the `Python Packaging User Guide <https://packaging.python.org>`__ to reduce duplicated effort across installers for covering this topic.
+
+Rejected Ideas
+==============
+
+Do not specify a name for the virtual environment directory
+-----------------------------------------------------------
+
+Using a consistent name for the virtual environment directory is important for a few reasons:
+
+1. It makes it easier for users to find the virtual environment directory, and to activate it.
+2. It removes a decision point for new users, since they do not need to decide on a name for the virtual environment directory.
+3. It creates a clear convention within the ecosystem, which makes it easier for users to find documentation.
+4. It ensures consistency across different tools, so that differences in the error message of different tools do not confuse users.
+
+Use a different name for the virtual environment directory
+----------------------------------------------------------
+
+Functionally, the directory name does not matter much as long as there is a single consistent suggestion.
+
+The name ``.venv`` was picked since it:
+
+1. does not conflict with any valid Python import name
+2. does not conflict ``venv`` module in the standard library
+3. has pre-existing usage in the Python community
+4. has support for auto-detection in common text editors
+5. can be typed without modifier keys on common keyboard layouts
+
+Do not couple tooling behaviour with a Python version
+-----------------------------------------------------
+
+This PEP creates a coupling between the behaviour of installers and the Python version, enforcing a behaviour change in installers based on a Python version.
+
+This is already a rollout mechanism being used for behaviour changes in the installation tooling. For example, ``pip`` on Python 3.11 will use ``importlib.metadata`` instead of ``pkg_resources`` for parsing/fetching package metadata, and ``sysconfig`` instead of ``distutils.sysconfig`` for getting the paths to unpack wheels into.
+
+The difference with those cases is that they're supposed to be largely transparent to end users. This PEP is proposing a behaviour change that is not transparent to end users, and requires them to take action.
+
+The primary benefit of this is that it allows for redistributors to adapt their tooling in time for the new Python version and provides a clear and consistent point for change across the ecosystem. It also puts a clear deadline on when the default behaviour will consistently require a virtual environment by default (once Python 3.12 goes end-of-life).
+
+The primary issue with this approach is that it enforces a behaviour change on users when they upgrade to a new Python version, which can hamper the adoption of a new Python version. However, this is a migration/upgrade for existing users and it is a common expectation that *some* changes will be needed for migration/upgrades.
```

Comment:
```suggestion
The primary issue with this approach is that it enforces a behaviour change on users when they upgrade to a new Python version, which can hamper the adoption of a new Python version. However, this is a migration/upgrade for existing users, and it is a common expectation that *some* changes will be needed for migration/upgrades.
```

Somehow missed this missing comma before, sorry

Reply:
This feels wrong to me as a reader; so skipping it. :)

Reply:
As above—it sounds wrong to me as a technical writer and copyeditor, and the same grammar applies—these are two independent clauses separated by a coordinating conjunction, so they need a comma. Particularly so since the second clause repeats a phrase ("migration/upgrades") from the previous clause, and the distance created by the comma also helps make this repetition feel less jarring and more intentional.

Also, fixed a plural/singular agreement error.

That being said, same as above applies—it's your PEP :)

Reply:
Let's leave this as-is.